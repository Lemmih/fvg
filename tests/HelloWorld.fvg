main : Svg
main =
  <svg>
    <text "font-size"="35" > "Hello World" </text>
  </svg>
