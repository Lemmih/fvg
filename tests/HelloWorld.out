 <svg > <text font-size="35" >Hello World </text>  </svg> 